src/main/webapp/dataset.xhtml

```diff
@@ -1607,6 +1629,107 @@
+                                        <p>#{bundle['license.custom']}</p>
+                                    </ui:fragment>
+                                    <ui:fragment rendered="#{!empty DatasetPage.workingVersion.termsOfUseAndAccess.license}">
+                                        <p>
+                                            <img src="#{DatasetPage.workingVersion.termsOfUseAndAccess.license.iconUrl}" title="#{DatasetPage.workingVersion.termsOfUseAndAccess.license.shortDescription}" style="display:none"  onload="this.style.display='inline'" />
+                                            <a href="#{DatasetPage.workingVersion.termsOfUseAndAccess.license.uri}" target="_blank">#{DatasetPage.workingVersion.termsOfUseAndAccess.license.name}</a>
+                                        </p>
+                                    </ui:fragment>
+                                </div>
+                            </div>
+                            <ui:fragment rendered="#{!empty DatasetPage.workingVersion.termsOfUseAndAccess.license}">
+                                <div class="form-group" jsf:rendered="#{!empty DatasetPage.workingVersion.termsOfUseAndAccess.license.shortDescription}">
+                                    <label for="metadata_Terms" class="col-sm-3 control-label">
+                                        #{bundle['dataset.publish.terms.description']}
+                                    </label>
+                                    <div class="col-sm-9">
+                                        <h:outputText value="#{DatasetPage.workingVersion.termsOfUseAndAccess.license.shortDescription}"/>
+                                    </div>
+                                </div>
+                            </ui:fragment>
+                            <ui:fragment rendered="#{empty DatasetPage.workingVersion.termsOfUseAndAccess.license}">
+                                <div class="form-group" jsf:rendered="#{!empty DatasetPage.workingVersion.termsOfUseAndAccess.termsOfUse}">
+                                    <label for="metadata_TermsUse" class="col-sm-3 control-label">
+                                        #{bundle['file.dataFilesTab.terms.list.termsOfUse.termsOfUse']}
+                                    </label>
+                                    <div class="col-sm-9">
+                                        <p:inputTextarea value="#{MarkupChecker:sanitizeBasicHTML(DatasetPage.workingVersion.termsOfUseAndAccess.termsOfUse)}" readonly="true" rows="3" styleClass="form-control" style="border:none"/>
+                                   </div>
+                                </div>
+                                <div class="form-group" jsf:rendered="#{!empty DatasetPage.workingVersion.termsOfUseAndAccess.confidentialityDeclaration}">
+                                    <label for="confidentialityDeclaration" class="col-sm-3 control-label">
```

Comment:
It may be clearer when I see the UI, but - since these aren't new fields - where were they before? I don't see these deleted from other .xhtml code as I'd expect if you were moving these here.

Reply:
UI works, but I'm still not sure if these now exist somewhere else as well (and they should be removed from there.) - will check.

Reply:
Ah - these are adds to the publish dialog where neither the license or terms showed previously.